### PR TITLE
Getting rid of depracated grpc function calls

### PR DESCRIPTION
**Pull request by iwankgb**

### CODE REVIEW

**Comment by iwankgb** [comment]
@dashpole I tested it locally against containerd instance and all the stats seemed to be available as expected.

File: container/containerd/client.go [hunk 1]
```diff
@@ -62,29 +64,37 @@ func Client(address, namespace string) (ContainerdClient, error) {
...
 			grpc.WithUnaryInterceptor(unary),
 			grpc.WithStreamInterceptor(stream),
 		)
 
-		conn, err := grpc.Dial(dialer.DialAddress(address), gopts...)
+		ctx, cancel := context.WithTimeout(context.Background(), connectionTimeout)
+		conn, err := grpc.DialContext(ctx, dialer.DialAddress(address), gopts...)
 		if err != nil {
 			retErr = err
+			cancel()
```
Comment (dashpole) [inline container/containerd/client.go:89]: should we just `defer cancel()` after the `context.WithTimeout(...` call?

Comment (iwankgb) [reply container/containerd/client.go:89]: Sounds legit.

File: container/containerd/client.go [hunk 2]
```diff
@@ -48,6 +49,7 @@ var ctrdClient ContainerdClient = nil
 
 const (
 	maxBackoffDelay   = 3 * time.Second
+	baseBackoffDelay  = 100 * time.Millisecond
```
Comment (dashpole) [inline container/containerd/client.go:52]: Where did this number come from?

**Comment by dashpole** [comment]
Just a few nits...

File: container/containerd/client.go (hunk 2 above)
Comment (iwankgb) [reply container/containerd/client.go:52]: I had to set it to some meaningful value and I thought that 100ms might make sense.

**Comment by iwankgb** [comment]
@googlebot I consent.

**Comment by iwankgb** [comment]
@dashpole my comment at googlebot seems to be ignored.

**Comment by dashpole** [comment]
It looks like it thinks the first three commits came from a different account

**Comment by iwankgb** [comment]
I guess that it might be caused be me using another email address. All commits use same address now :)

**Comment by iwankgb** [comment]
@dashpole the last remaining comment is yours about value of `baseBackoffDelay`. I decided to set to 100ms but I'm open to change it to any other value. My gut feeling is that this one isn't that bad. Take a look at [the docs](https://github.com/grpc/grpc-go/blob/master/backoff/backoff.go#L30), please.

**Comment by dashpole** [comment]
sorry, it just fell off my radar

**Pull request merged**